src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -100,6 +100,16 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
+            if (InternalTestHooks.ActivateSleepForStoppingTest)
+            {
+                // We'll wait the verbose stream marker in tests and then call Stop().
+                // The cmdlet is very fast and likely to end before the test sees the marker
+                // so we have to slow down it by Sleep(50).
+                // We can not sleep for a long time because Stop() doesn't work on sleeping runspace.
+                WriteVerbose("ConvertTo-Json started");
+                System.Threading.Thread.Sleep(50);
```

Comment:
I would be happier if this value was in the testhook (ala TestStopComputerResults). This would enable us to change the duration easily, and even specific to a particular test.

Reply:
Done.